package.json

```diff
@@ -52,12 +52,10 @@
     "publish": "npm-run-all --serial publish:*",
     "publish:packages": "yarn changeset publish --public",
     "publish:storybook": "build-storybook && gh-pages --dist ./storybook-static",
-    "publish:website": "yarn build && (cd website && yarn build && yarn run publish)",
```

Comment:
we're still referencing this script in `release.yml`

Reply:
So strange how this didn't come up when I did a find-all for the term `website`. Good catch!